demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -881,19 +882,237 @@ def test_collect_playbooks_dependencies_skip_unavailable(self, id_set):
+                                        "endCol": 2,
+                                        "fieldId": "indicator_example",
+                                        "height": 22,
+                                        "id": "example",
+                                        "index": 0,
+                                        "sectionItemType": "field",
+                                        "startCol": 0
+                                    }
+                                ],
+                                "maxW": 3,
+                                "minH": 1,
+                                "minW": 1,
+                                "moved": "false",
+                                "name": "New Section",
+                                "static": "false",
+                                "w": 1,
+                                "x": 0,
+                                "y": 0
+                            }
+                        ],
+                        "type": "custom"
+                    }
+                ]
+            },
+            "name": "example",
+            "system": "false",
+            "version": -1,
+            "fromVersion": "6.0.0",
+            "description": ""
+        }
+
+        pack1 = repo.create_pack('pack1')
+        pack2 = repo.create_pack('pack2')
+        pack3 = repo.create_pack('pack3')
+        pack4 = repo.create_pack('pack4')
+
+        pack1.create_incident_field('example', incident)
+        pack2.create_indicator_field('example', indicator)
+        pack3.create_layoutcontainer('example', incident_layout)
+        pack4.create_layoutcontainer('example', indicator_layout)
+
+        with ChangeCWD(repo.path):
+            # Circle froze on 3.7 dut to high usage of processing power.
+            # pool = Pool(processes=cpu_count() * 2) is the line that in charge of the multiprocessing initiation,
+            # so changing `cpu_count` return value to 1 still gives you multiprocessing but with only 2 processors,
+            # and not the maximum amount.
+            import demisto_sdk.commands.common.update_id_set as uis
+            mocker.patch.object(uis, 'cpu_count', return_value=1)
+            PackDependencies.find_dependencies(pack, silent_mode=True)
+            dependencies = list(repo.packs[pack_in_repo].pack_metadata.read_json_as_dict().get('dependencies').keys())
+            assert dependencies == [expected_dependencies]
+
+    def test_collect_incident_layout_dependencies(self, id_set):
+        """
+        Given
+            - A layout of type incident entry in the id_set.
 
         When
             - Building dependency graph for pack.
 
         Then
             - Extracting the packs that the layout depends on.
         """
-        expected_result = {("FeedMitreAttack", True), ("PrismaCloudCompute", True), ("CommonTypes", True),
-                           ("CrisisManagement", True)}
+        expected_result = {("PrismaCloudCompute", True)}
```

Comment:
Why did you change this test?

Reply:
Because the rest of the packs has indicator_fields and the layout is of type incident, so the layout is not depend on them